### PR TITLE
Add packaged-as-an-iexpress-self-extracting-archive

### LINKED ISSUE #185: packaged-as-an-iexpress-installer
## Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your rule idea isn't already filed: [search](https://github.com/fireeye/capa-rules/issues?q=is%3Aissue+is%3Aopen+)

## Summary

It'd be cool to have a rule that detects EXEs created with `iexpress.exe`

## Examples

52e25bdd600695cfed0d4ee3aca4f121bfebf0de889593e6ba06282845cf39ea

## Features

Is it possible to match on the EXE manifest with capa?
```
<!-- Copyright (c) Microsoft Corporation -->
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">

  <assemblyIdentity version="5.1.0.0"
     processorArchitecture="amd64"
     name="wextract"
     type="win32"/>
  
  <description>IExpress extraction tool</description>
...
```

Otherwise, matching on the two strings shown below in the same function scope should be a good indicator:
image

From: https://analyze.intezer.com/analyses/6e1d2029-a4ac-477a-8187-03670732b745/families/0a29c6e3-2b86-4e82-b034-57a08b011ad6

## Additional context

N/A

### Namespace
executable/installer/iexpress

### References
https://en.wikipedia.org/wiki/IExpress

### Other rule meta information
T1207 - Obfuscated Files or Information

### PR DESCRIPTION
**Pull request by recvfrom**
Fix #185

### CODE REVIEW

**Review by mr-tz — APPROVED** [review]
looks good, thank you!

File: executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    name: packaged as an IExpress self-extracting archive
+    namespace: executable/installer/iexpress
+    author: '@recvfrom'
+    scope: file
+    references:
+      - https://en.wikipedia.org/wiki/IExpress
+    examples:
+      - ac742739cae0d411dfcb78ae99a7baee:0xA4C0 # wextract_cleanup%d
+      - ac742739cae0d411dfcb78ae99a7baee:0xA488 # Software\Microsoft\Windows\CurrentVersion\RunOnce
+      - ac742739cae0d411dfcb78ae99a7baee:0x34BA2 # '  <description>IExpress extraction tool</description>'
```
Comment (williballenthin) [inline executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml:12]: i'm not sure that we have other file scope examples as file offsets, but this makes sense to me

File: executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: packaged as an IExpress self-extracting archive
+    namespace: executable/installer/iexpress
```
Comment (williballenthin) [inline executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml:4]: this mirrors `executable/installer/installshield`, good

https://github.com/fireeye/capa-rules/blob/1ff994f7916d66e39b4b5b8dbb310d0e0b051f7f/nursery/packaged-as-an-installshield-installer.yml#L4

Comment (williballenthin) [reply executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml:4]: we're starting to see the compiler vs packager split a bit (e.g. gcc is definitely a compiler, exe4j is right now under "compiler", and iexpress is definitely not a compiler and "installer" makes sense). no changes recommended, just highlighting this sorta annoying question.

**Review by williballenthin — APPROVED** [review]

**Comment by williballenthin** [comment]
thank you!

**Pull request merged**